examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,20 +3,21 @@
  * Licensed under the MIT License.
  */
 
-import { SchemaBuilder, Typed, leaf } from "@fluid-experimental/tree2";
+import { ProxyNode, SchemaBuilder } from "@fluid-experimental/tree2";
 
-const builder = new SchemaBuilder({ scope: "inventory app" });
+const builder = new SchemaBuilder({
+	scope: "com.contoso.app.inventory",
+});
 
-export const part = builder.struct("Contoso:Part-1.0.0", {
-	name: leaf.string,
-	quantity: leaf.number,
+export const part = builder.struct("Part", {
+	name: builder.string,
+	quantity: builder.number,
 });
 
-export const inventory = builder.struct("Contoso:Inventory-1.0.0", {
-	parts: builder.sequence(part),
+export const inventory = builder.struct("Inventory", {
+	parts: builder.fieldNode("List<Part>", builder.sequence(part)),
```

Comment:
List is checked in! It still needs a name for now, though.

```suggestion
	parts: builder.list("List<Part>", part),
```

Reply:
Why does list need a name?

Reply:
Looks like Craig landed structural naming too!
```
const fullName = structuralName("List", types);
```

Reply:
Nevermind - I missed the first overload when I was looking at the list function in my repo. I remember seeing that PR but I forgot about it. @CraigMacomber is checking in so much code that I can't keep up :D